src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -53,14 +53,20 @@ public ApplicationTheme RequestedTheme
 		public event SuspendingEventHandler Suspending;
 #pragma warning restore CS0067 // The event is never used
 
-		public event UnhandledExceptionEventHandler UnhandledException;
-
-		[NotImplemented]
+		public event UnhandledExceptionEventHandler UnhandledException;
+
+#if __ANDROID__
+		public void Exit()
+		{
+			Android.OS.Process.KillProcess(Android.OS.Process.MyPid());
+		}
+#else
```

Comment:
Please move the Android specific code to the `Application.Android.cs` file

Reply:
But what with [unimplemented] for all other platforms? It is defined in Application.cs here, not in Generated ("// Skipping already declared method Windows.UI.Xaml.Application.Exit()") - I don't know why. Should I move Exit to Generated file?

Reply:
Use `#if !__ANDROID__` for the not implemented method in this file.